modules/customizing-cli-downloads.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * web-console/customizing-the-web-console.adoc
+
+[id="creating-custom-CLI-downloads_{context}"]
+= Customizing CLI downloads
+
+You can configure links for downloading the CLI with custom link text and URLs,
+which can point directly to file packages or to an external page that provides
+the packages.
+
+.Procedure
+
+. Navigate to *Administration* -> *Custom Resource Definitions*.
+
+. Select *ConsoleCLIDownload* from the list of Custom Resource Definitions (CRDs).
+
+. Click the *YAML* tab, and then make your edits.
+
```

Comment:
Here is an example that can be `oc create -f <file>.yaml` or can be pasted in the console as you stated:

```yaml
apiVersion: console.openshift.io/v1
kind: ConsoleCLIDownload
metadata:
  name: example-cli-download-links-for-foo.
spec:
  description: |
    This is an example of download links for foo
  displayName: example-foo
  links:
    - href: 'https://www.foo-for-linux.com'
      name: foo for linux
    - href: 'https://www.foo-for-mac.com'
      name: foo for mac
    - href: 'https://www.foo-for-win.com'
      name: foo for windows
```